#! /bin/sh /usr/share/dpatch/dpatch-run
## makefile-am-manpages.dpatch by  <blair@ctrlpclnx.nscl.msu.edu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Generate manpages rather than 0-byte files

@DPATCH@

--- epicstcl-2.0-001/Makefile.am	2009-03-17 11:06:48.000000000 -0400
+++ epicstcl-2.0-001-new/Makefile.am	2009-03-17 15:43:39.000000000 -0400
@@ -127,12 +129,13 @@
 	$(DVIPDF)     manual.dvi
 	mv manual.pdf epics.pdf
 
+.xml.1tcl:
+	docbook-to-man $< > $@
 
+.xml.3tcl:
+	docbook-to-man $< > $@
 
-$(MANPAGES): $(WEBXMLFILES)
-	$(MANDOCBOOK) manual.xml
-	touch $(MANPAGES)
-
+$(MANPAGES): $(XMLFILES)
 
 man_MANS = $(MANPAGES)
